From b194b56ece660c95064b471dad9b356098b18972 Mon Sep 17 00:00:00 2001
From: Janne Blomqvist <janne.blomqvist@aalto.fi>
Date: Wed, 23 Jan 2019 22:25:18 +0200
Subject: [PATCH] Support stateful provisioning to NVME devices

In order to do stateful provisioning to NVME devices, the NVME host
drivers need to be added to the bootstrap image. Also, as partitions
on NVME devices are named by appending 'pN' to the device name, add an
example filesystem config demonstrating how to do this.

Signed-off-by: Janne Blomqvist <janne.blomqvist@aalto.fi>
---
diff --git a/vnfs/etc/bootstrap.conf b/vnfs/etc/bootstrap.conf
index fe74fbe..893815d 100644
--- a/vnfs/etc/bootstrap.conf
+++ b/vnfs/etc/bootstrap.conf
@@ -5,6 +5,7 @@
 drivers += kernel/drivers/net/, kernel/drivers/scsi/, kernel/drivers/ata/
 drivers += kernel/drivers/virtio/, kernel/drivers/message/, kernel/drivers/md/
 drivers += kernel/drivers/block/, kernel/drivers/usb/host/, kernel/drivers/firmware/efi/
+drivers += kernel/drivers/nvme/host/
 drivers += nfs, nfsd, nfs_common, nfsv4
 drivers += fuse, ext2, ext3, ext4
 drivers += fat, vfat, nls_cp437, nls_ascii, efivarfs
